docs/deployment/feature_gates.md

```diff
@@ -69,5 +70,6 @@ A *General Availability* (GA) feature is also referred to as a *stable* feature.
 * `HVPAForShootedSeed`  enables simultaneous horizontal and vertical scaling in shooted Seed clusters.
 * `ManagedIstio` enables a Gardener-tailored [Istio](https://istio.io) in each Seed cluster. Disable this feature if Istio is already installed in the cluster. Istio is not automatically removed if this feature is disabled. See the [detailed documentation](../usage/istio.md) for more information.
 * `APIServerSNI` enables only one LoadBalancer to be used for every Shoot cluster API server in a Seed. Enable this feature when `ManagedIstio` is enabled or Istio is manually deployed in Seed cluster. See [GEP-8](../proposals/08-shoot-apiserver-via-sni.md) for more details.
-* `MountHostCADirectories` enables mounting common CA certificate directories in the Shoot API server pod that might be required for webhooks or OIDC. 
+* `MountHostCADirectories` enables mounting common CA certificate directories in the Shoot API server pod that might be required for webhooks or OIDC.
 * `SeedChange` enables updating the `spec.seedName` field during shoot validation from a non-empty value in order to trigger shoot control plane migration.
+* `SeedKubeScheduler` adds custom `kube-scheduler` in `gardener-kube-scheduler` namespace. It schedules [pods with scheduler name](../concepts/seed-admission-controller.md#mutating-webhooks) `gardener-kube-scheduler` on Nodes with higher resource utilization. It requires Seed cluster with kubernetes version `1.17` or higher.
```

Comment:
Is there a good reason for not putting it into the `garden` namespace like all the other system components we deploy?

Reply:
Putting everything in one namespace offers a very easy way to clean resources when disabling the feature. The extensions are already in a separate namespaces already.

Reply:
Why not use extensions to deploy it?

Reply:
I would need a new controller as the version of the `kube-scheduler` must match the Seed API server version.

Reply:
> Putting everything in one namespace offers a very easy way to clean resources when disabling the feature. The extensions are already in a separate namespaces already.

Isn't this also the case when using a ManagedResource for that?
TBH, I would prefer to have this consistent with other system components we deploy for example `etcd-druid` and `gardener-seed-admission-controller`. So deploying it via a MR into the `garden` namespace.
Sorry, if I missed something here. @rfranzke you resolved this, did you change your mind?

Reply:
Yes, I don't care too much about it and I'm okay with the solution as it is now. We also already deploy the istio components and the extensions into separate namespaces.